oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/KafkaFetcherHandlerRegister.java

```diff
@@ -132,16 +157,21 @@ public void start() {
     @Override
     public void run() {
         while (true) {
-            ConsumerRecords<String, Bytes> consumerRecords = consumer.poll(Duration.ofMillis(500L));
-            if (!consumerRecords.isEmpty()) {
-                Iterator<ConsumerRecord<String, Bytes>> iterator = consumerRecords.iterator();
-                while (iterator.hasNext()) {
-                    ConsumerRecord<String, Bytes> record = iterator.next();
-                    handlerMap.get(record.topic()).handle(record);
+            try {
```

Comment:
I don't think we still need this single thread pool when we introduce the thread pool. @zifeihan 
Could we use the same one?

https://github.com/apache/skywalking/blob/b79436e798546102781b9e57f1116eb74113a7d8/oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/KafkaFetcherHandlerRegister.java#L154

Reply:
@dmsolr hello, kafka consumer poll data is fast, and each partition can only have one thread to consume. In our stress testing process, data pulled by Kafka is not a performance bottleneck, but trace data analysis is the performance bottleneck. So I think it's better to pull the data as it is.

Reply:
@dmsolr Modified to use the same thread pool to poll Kafka data and process data.